core/modules/syncer.js

```diff
@@ -202,7 +209,15 @@ Syncer.prototype.syncFromServer = function() {
 			clearTimeout(this.pollTimerId);
 			this.pollTimerId = null;
 		}
+		
 		this.syncadaptor.getSkinnyTiddlers(function(err,tiddlers) {
+			//get an array of the titles being updated
+			var titles = tiddlers.map(e => e.title);
```

Comment:
I think TW doesn't use this type of function calls in the core. ... @Jermolene .. do you allow to use "arrow functions"?

Reply:
Can't use arrow functions because TW still supports ES5 browsers. Unless we run things through a transpiler. (I run my own plugins through babel but I don't think the core does).

Reply:
Yeah, that is the code that I didn't know if it is correct or not in the handling of deletions. If the server and client sync filters are different then this will cause problems, but if they are a perfect snapshot of each other then we are fine.